components/merkledb/README.md

```diff
@@ -0,0 +1,84 @@
+# Exonum MerkleDB
+
+[![Travis Build Status](https://img.shields.io/travis/exonum/exonum/master.svg?label=Linux%20Build)](https://travis-ci.com/exonum/exonum)
+[![Docs.rs](https://docs.rs/exonum-merkledb/badge.svg)](https://docs.rs/exonum-merkledb)
+![rust 1.34.0+ required](https://img.shields.io/badge/rust-1.34.0+-blue.svg?label=Required%20Rust)
+[![License: Apache-2.0](https://img.shields.io/github/license/exonum/exonum.svg)](https://github.com/exonum/exonum/components/merkledb/blob/master/LICENSE)
+
+Exonum MerkleDB is a persistent storage implementation based on RocksDB
+which provides APIs to work with merkelized data structures.
+
+## Available database objects
+
+- `Entry` is a specific index that stores only one value. Useful for global
+  values, such as configuration. Similar to a combination of `Box` and
+  `Option`.
+- `ListIndex` is a list of items stored in the sequential order. Similar to
+  `Vec`.
+- `SparseListIndex` is a list of items stored in the sequential order. Similar
+  to `ListIndex`, but may contain indices without elements.
+- `MapIndex` is a map of keys and values. Similar to `BTreeMap`.
+- `ProofListIndex` is a Merkelized version of `ListIndex` that supports
+  cryptographic proofs of existence and is implemented as a Merkle tree.
+- `ProofMapIndex` is a Merkelized version of `MapIndex` that supports cryptographic
+  proofs of existence and is implemented as a binary Merkle Patricia tree.
+- `KeySetIndex` and `ValueSetIndex` is a set of items, similar to `BTreeSet` and
+  `HashSet`.
+
+## Usage
+
+Include `exonum-merkledb` as a dependency in your `Cargo.toml`:
+
+```toml
+[dependencies]
+exonum = "0.11.0"
```

Comment:
Do we really need `exonum` dependency in this case ?

Reply:
I think that we must strongly avoid of `exonum` dependency here, in my mind external developers have to know, that they can use exonum-merkledb in their projects that don't even linked with exonum or other blockchains.
Moreover they have possibility to implement new blockchains from scratch on top of `exonum-merkledb`.